src/source/src/manager.rs

```diff
@@ -135,7 +134,7 @@ impl SourceManager for MemSourceManager {
         );
         let row_id_index = info.row_id_index as usize;
 
-        let source = SourceImpl::Connector(ConnectorSource {
+        let source = SourceImpl::ParallelConnector(ParallelConnectorSource {
```

Comment:
Then why do `SourceImpl::Connector` still needed? Can we just use `SourceImpl::Connector` to represent parallel connector?

Reply:
removed